src/System.Private.CoreLib/shared/System/Numerics/ConstantHelper.tt

```diff
@@ -14,6 +14,7 @@
 <#@ import namespace="System.Runtime.InteropServices" #>
 <#@ include file="GenerationConfig.ttinclude" #><# GenerateCopyrightHeader(); #>
 
+#nullable enable
```

Comment:
What is this tt file for? Just curious.

Reply:
It's a template that autogenerates the similarly named .cs file.